e2e/playwright/test-network-and-connection-issues.spec.ts

```diff
@@ -14,8 +14,10 @@ test.describe('Test network related behaviors', () => {
     'simulate network down and network little widget',
     { tag: '@skipLocalEngine' },
     async ({ page, homePage }) => {
-      const networkToggleConnectedText = page.getByText('Connected')
-      const networkToggleWeakText = page.getByText('Network health (Weak)')
+      const networkToggleConnectedText = page.getByText(
+        'Network health (Strong)'
+      )
+      const networkToggleWeakText = page.getByText('Ok')
```

Comment:
Should this one be updated as well?

```suggestion
      const networkToggleWeakText = page.getByText('Network health (Ok)')
```